openstudio-extension.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'bundler', '~> 1.9'
+  spec.add_dependency 'bundler', '~> 2.0'
```

Comment:
@jmarrec any reason to not make this `~> 2.1`

Reply:
Not to me personally. I think it'll work with any 2.x so perhaps not worth forcing other people to upgrade, but 2.1 would work for me for openstudio

Reply:
Okay, then I think we should use 2.1. I mean, ~> 2.0 would resolve to 2.1.x anyway, but it would be nice to use at lease 2.1 and greater.

Reply:
another piece of concern is that 2.x requires ruby > 2.2... which right now I think we are unable to support, correct?

Reply:
Well, NREL/OpenStudio's develop branch is at 3.0.0-beta1, which uses Ruby 2.5.5.
The gem repos should perhaps keep a 2.9.X-LTS branch like we do on various repos, eg: https://github.com/NREL/openstudio-gems/tree/2.9.X-LTS

@tijcolem thoughts please?

Reply:
Can we freeze gems that support 2.9-LTX so it only uses these versions (below)?  2.9-LTX is supposed to be bug fix only and not support new features. We should bump the version of the gems once we update to the new bundler. 

2.9-LTX support gems (with bundler 1.17.x) 
gem 'openstudio-extension', '= 0.1.2'
gem 'openstudio-workflow', '= 1.3.4'
gem 'openstudio-standards', '= 0.2.10'
gem 'openstudio_measure_tester', '= 0.1.7' 

3.0.x- support gems (with bundler 2.1.x) 
gem 'openstudio-extension', '= ?'
gem 'openstudio-workflow', '= ?'
gem 'openstudio-standards', '= ?'
gem 'openstudio_measure_tester', '= ?'

Reply:
I think this makes more sense... we can bump everything (openstudio-*.gem) to 0.2.x which support Ruby 2.5.5 and bundler 2.1.x. The previous versions 0.1.x will work for ruby 2.2.4 and bundler 1.17.x